base/ltfssbas.dtx

```diff
@@ -35,7 +35,7 @@
 %
 %
 \ProvidesFile{ltfssbas.dtx}
-             [2025/02/20 v3.2p LaTeX Kernel (NFSS Basic Macros)]
+             [2025/03/06 v3.2q LaTeX Kernel (NFSS Basic Macros)]
```

Comment:
Perhaps this should still be 2025/03/05, not 06. Could be changed at last minute.
```suggestion
             [2025/03/05 v3.2q LaTeX Kernel (NFSS Basic Macros)]
```